ctx.go

```diff
@@ -848,9 +848,9 @@ func (c *Ctx) Path(override ...string) string {
 	return c.path
 }
 
-// Protocol contains the request protocol string: http or https for TLS requests.
+// Scheme contains the request scheme: http or https for TLS requests.
 // Use Config.EnableTrustedProxyCheck to prevent header spoofing, in case when your app is behind the proxy.
-func (c *Ctx) Protocol() string {
+func (c *Ctx) Scheme() string {
```

Comment:
@abhi12299 can you look c.Request().URI().Scheme(), it could return `http` or `https`

Reply:
@balcieren I haven't changed the behavior of the old `Protocol` function.. just renamed it to `Scheme`. So it works just like it used to previously.